hack/deploy-hub-local/lab-dns-internal.sh

```diff
@@ -40,7 +40,11 @@ resolv-file=/etc/resolv.upstream.conf
 # Hub Cluster
 address=/.apps.test-ci.alklabs.local/192.168.150.252
 address=/api.test-ci.alklabs.local/192.168.150.253
-address=/api-int.test-ci.alklabs.local/192.168.150.253" > /etc/NetworkManager/dnsmasq.d/00-test-ci.conf
+address=/api-int.test-ci.alklabs.local/192.168.150.253
+# Spoke Cluster
```

Comment:
Should we separate the Spoke DNS assignation in the Spoke side?

Reply:
same